libs/core/src/lib/table/table.component.scss

```diff
@@ -22,4 +22,4 @@
 	&--no-wrap {
 		white-space: nowrap;
 	}
-}
+}
```

Comment:
What is changed in this file?

Reply:
No changes to this file. I started out adding a style to this file and then removed it.

Reply:
I found out it was missing a new line character at the end of the file. I put it back in.